tiledb/sm/rest/curl.h

```diff
@@ -185,6 +186,25 @@ class Curl {
   /** Extra headers to attach to each request. */
   std::unordered_map<std::string, std::string> extra_headers_;
 
+  /** Response headers attached to each response. */
+  std::unordered_map<std::string, std::string> res_headers_;
+
+  /**
+   * This callback function gets called by libcurl as soon as a header has been
+   * received. libcurl buffers headers and delivers only "full" headers, one by
+   * one, to this callback. This callback should return the number of bytes
+   * actually taken care of if that number differs from the number passed to
+   * your callback function, it signals an error condition to the library
+   *
+   * @param res_data points to the delivered data
+   * @param tSize the size of that data is size multiplied with nmemb
+   * @param tCount number of bytes actually taken care of
```

Comment:
Although curl may use hungarian notation for naming variables, we do not. Let's rename `tSize` to `size` and `tCount` to `count`.